crypto/rand/randfile.c

```diff
@@ -12,6 +12,7 @@
 #include <errno.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <unistd.h>
```

Comment:
Did anyone already point out that Windows doesn't have `unistd.h`?

Reply:
> OpenSSL doesn't make an assumption that POSIX interface is always an option.